stripe/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -69,7 +84,10 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
     /**
      * This should be initialized from the starter args, and then from that
      * point forward it will be the last valid card seen or entered in the add card view.
-     * In contrast to selection, this field will not be updated by the list fragment.
+     * In contrast to selection, this field will not be updated by the list fragment.  On the
```

Comment:
Mega nit (potential holy war): let's use single space for consistency.

Reply:
Did a find replace everywhere, and updated my mac keyboard settings :D